tests/components/switch/test_device_trigger.py

```diff
@@ -162,58 +173,59 @@ def calls(hass):
     hass.states.async_set(ent1.entity_id, STATE_OFF)
     await hass.async_block_till_done()
     assert len(calls) == 1
-    assert calls[0].data["some"] == "turn_off device - {} - on - off - None".format(
-        ent1.entity_id
-    )
+    assert calls[0].data[
+        "some"] == "turn_off device - {} - on - off - None".format(
+            ent1.entity_id)
 
     hass.states.async_set(ent1.entity_id, STATE_ON)
     await hass.async_block_till_done()
     assert len(calls) == 2
-    assert calls[1].data["some"] == "turn_on device - {} - off - on - None".format(
-        ent1.entity_id
-    )
+    assert calls[1].data[
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4636028)